RecoMET/METPUSubtraction/plugins/PFMETProducerMVA.cc

```diff
@@ -16,12 +16,13 @@ PFMETProducerMVA::PFMETProducerMVA(const edm::ParameterSet& cfg)
   srcPFCandidatesView_ = consumes<reco::CandidateView>(cfg.getParameter<edm::InputTag>("srcPFCandidates"));
   srcVertices_     = consumes<reco::VertexCollection>(cfg.getParameter<edm::InputTag>("srcVertices"));
   vInputTag srcLeptonsTags = cfg.getParameter<vInputTag>("srcLeptons");
+  edm::InputTag srcRho = ( cfg.exists("srcRho") ) ? cfg.getParameter<edm::InputTag>("srcRho") : (edm::InputTag("fixedGridRhoFastjetAll"));
+  consumes<double>(srcRho);
```

Comment:
Hi,
that's right, it's not used directly. But the L1FastjetCorrector uses it while it does not do the consumes call itself. So without it, the plugin won't work.
https://github.com/cms-sw/cmssw/blob/CMSSW_7_6_X/JetMETCorrections/Algorithms/src/L1FastjetCorrector.cc
That's also the reason I don't want to force users to specify something in the python config: It would suggest to them this configurable would actually change physics.